drivers/SmartThings/zwave-lock/fingerprints.yml

```
@@ -294,6 +294,13 @@ zwaveManufacturer:
     productType: 0x0001
     productId: 0x0001
     deviceProfileName: base-lock
+  # Philips
+  - id: "541/2307/212"
+    deviceLabel: Philips
```

```suggestion
    deviceLabel: Philips Lock
```
This was probably intended, right?